src/joystick/apple/SDL_mfijoystick.m

```diff
@@ -1826,16 +1828,18 @@ static void GetAppleSFSymbolsNameForElement(GCControllerElement *element, char *
                 case SDL_GAMEPAD_BUTTON_MISC1:
                     GetAppleSFSymbolsNameForElement(elements[GCInputDualShockTouchpadButton], elementName);
                     break;
-                case SDL_GAMEPAD_BUTTON_PADDLE1:
+                case SDL_GAMEPAD_BUTTON_RIGHT_PADDLE1:
                     GetAppleSFSymbolsNameForElement(elements[GCInputXboxPaddleOne], elementName);
                     break;
-                case SDL_GAMEPAD_BUTTON_PADDLE2:
+                case SDL_GAMEPAD_BUTTON_LEFT_PADDLE1:
+                    /* TODO: Is this right? SDL_gamepad.h says P2 is the lower right */
                     GetAppleSFSymbolsNameForElement(elements[GCInputXboxPaddleTwo], elementName);
                     break;
-                case SDL_GAMEPAD_BUTTON_PADDLE3:
+                case SDL_GAMEPAD_BUTTON_RIGHT_PADDLE2:
+                    /* TODO: Is this right? SDL_gamepad.h says P3 is the upper left */
                     GetAppleSFSymbolsNameForElement(elements[GCInputXboxPaddleThree], elementName);
                     break;
```

Comment:
I don't have any Apple hardware, but I think the current code might be wrong here, swapping P2 and P3. For now I've preserved existing behaviour.